### PR TITLE
Add JSON schemas for registrations and reports

### PR DESCRIPTION
**Pull request by apasel422**

### CODE REVIEW

**Review by csharrison — APPROVED** [review]
Mostly RS LGTM, I mostly skimmed / spot checked. LMK if you want a more careful review from both of us.

**Review by csharrison — APPROVED** [review]
Mostly RS LGTM, I mostly skimmed / spot checked. LMK if you want a more careful review from both of us.

File: schema/aggregatable_report.schema.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "properties": {
+    "aggregation_coordinator_identifier": {
+      "type": "string"
+    },
+    "aggregation_service_payloads": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "debug_cleartext_payload": {
```
Comment (linnan-github) [inline schema/aggregatable_report.schema.json:16]: This field is optional, shall we add default null?

File: schema/source.schema.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  "title": "Source",
+  "description": "An Attribution Reporting source registration",
+  "type": "object",
+  "properties": {
+    "aggregatable_report_window": {
+      "$ref": "#/$defs/signed_duration_in_seconds"
+    },
+    "aggregation_keys": {
+      "type": "object",
+      "additionalProperties": {
```
Comment (linnan-github) [inline schema/source.schema.json:13]: It's currently not in the spec, but shall we add the bytes limit in the schema? Similar for filters.

Comment (apasel422) [reply schema/source.schema.json:13]: It's not in the spec right now, so no.

**Pull request merged**